src/pages/groupComparison/LollipopTooltipCountInfo.tsx

```diff
@@ -0,0 +1,36 @@
+import { action, makeObservable, observable } from 'mobx';
+import { observer } from 'mobx-react';
+import { Mutation } from 'cbioportal-ts-api-client';
+import * as React from 'react';
+import {
+    formatPercentValue,
+    numberOfLeadingDecimalZeros,
+} from 'cbioportal-utils';
+import { AxisScale } from 'react-mutation-mapper';
+
+interface ILollipopTooltipCountInfoProps {
+    count: number;
+    mutations?: Mutation[];
+    axisMode?: AxisScale;
+}
+
+export const LollipopTooltipCountInfo: React.FC<ILollipopTooltipCountInfoProps> = ({
```

Comment:
Considering that this code is mostly taken from signaldb codebase we should also move this into `react-mutation-mapper` so that we can reuse it in signaldb to avoid duplication.